modules/acme/foobar/hooks/entrypointhook.go

```diff
@@ -18,10 +19,15 @@ type EntrypointHook struct {
 
 func (h EntrypointHook) Handle(
 	_ context.Context,
-	_ invocation.Context,
+	_ *invocation.ModuleContext,
 	payload hookstage.EntrypointPayload,
+	debug bool,
```

Comment:
I propose moving `debug` to `ModuleContext`

Reply:
debug param is overall for all modules, meanwhile ModuleContext is module specific. that's why adding a debug param to ModuleContext can be a bit deceptive in module configuration context. But I understand your point that it simplifies the hook's signature